loops_nomework/program_that_will_find_the_first_non_consecutive_number_in_a_list.py

```diff
@@ -0,0 +1,7 @@
+my_list = [1, 2, 3, 4, 6, 7, 8]
+filtered = []
+for element in range(0, len(my_list) - 1):
+    if my_list[element] + 1 != my_list[element + 1]:
+         print(my_list[element + 1])
+         break
```

Comment:
What if list is consecutive? You should add block else for loop
![image](https://github.com/OleksandrBochenko/test_repo_python/assets/51053483/7e0ad83f-6e9d-4387-8942-ae18a2399dd5)

Reply:
fixed